### ISSUE TITLE
Saving File Path

### ISSUE THREAD

**Issue by reeve567** [desophos/idleon-saver#40]
It'd be really handy if the app actually saved where my idleon exe is, since it's on a different drive I have to manually copy and paste the path every time, which can be a bother.  It already has a folder in appdata, so I figured it wouldn't be too much else to save a small config.  I'll open a pull request if I end up figuring it out.

**Comment by desophos** [comment]
I agree, this would be a great feature!

**Comment by reeve567** [comment]
Tried to get going on this but poetry has been a royal pain trying to update to the preview version :l

Did you encounter an issue with the file being in use when trying to update to preview?

**Comment by desophos** [comment]
I think I've run into that before. Are you using `poetry self update --preview`? You might need to run it from an admin terminal.

**Comment by reeve567** [comment]
Yeah, but once I tried running the poetry update it killed itself and I had to reinstall, so I'll try again after I move to Linux later this week, maybe that'll change something

**Comment by SmeepOG** [comment]
Agreed, my Steam folder is on E: and windows on C: - this seems to have been overlooked in the app
